pom.xml

```diff
@@ -15,7 +15,7 @@
 	<description>Demo project for Spring Boot</description>
 	<properties>
 		<java.version>11</java.version>
-		<dcsa.events.version>0.0.51</dcsa.events.version>
+		<dcsa.events.version>0.0.52</dcsa.events.version>
```

Comment:
This is dependant on even-core release

Reply:
as long the dev-bkg branch in event_core uses this version you are good, the  CI will adjust accordingly.